service.go

```diff
@@ -387,6 +387,19 @@ type Service interface {
 	Status() (Status, error)
 }
 
+//ConfigInfo is an optional interface which allows for certain information to be obtained.
+type ConfigInfo interface {
+	configpath() string
```

Comment:
`ConfigPath() (string, error)`

Reply:
I can't use that ConfigPath as it is already taken in most implementations. Also it doesn't have to be public anyways since it isn't directly called.